CardsDAO.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Linq;
+
+namespace Card_Game
+{
+    public class CardsDAO : ICardsDAO
+    {
+        private List<Card> cards = null;
+        private List<Card> Cards
```

Comment:
Small thing, but just to clarify. Here the Cards being private are a bit confusing to me, because from what I learned, private properties start with lowercase, right? Does it start with a capital letter because of the lazy initialization?

Reply:
Properties are always upper case, fields are lower case, their accessibility doesn't matter